### ISSUE TITLE
[Bug] Subscribe to multiple lists with same email

### ISSUE THREAD

**Issue by katiasmet** [barrelstrength/craft-sprout-lists#23]
### Description
When I subscribe to a different list with the same email, the email is just moved from one list to another. Is this intended behaviour?

Subscribing in the CP works (with the checkboxes).

I searched for a solution but can't find the problem :). It looks like the records are deleted and again created or smth. 

### Steps to reproduce

1. Make 2 lists
2. Subscribe with the same emailaddress to both lists

ElementID and ListHandle are different. Email is the same.

```html
<form class="c-form" method="post" accept-charset="utf-8">
  {{ csrfInput() }}
  <input type="hidden" name="action" value="sprout-lists/lists/subscribe">
  <input type="hidden" name="listHandle" value="{{ event.slug }}">
  <input type="hidden" name="elementId" value="{{ event.id }}">

   <input class="c-form__input" type="text" name="firstName">
   <input class="c-form__input" type="text" name="lastName">
   <input class="c-form__input" type="email" name="email">

  <button type="submit" name="button" class="c-btn c-btn--primary u-margin-top">
    Participate
  </button>
</form>
```

### Additional info

- Craft version: Craft Solo 3.0.28
- PHP version: 7.1.16
- Database driver & version: MySQL 5.7.20
- Plugins & versions: 1.0.0-beta.10 (spout lists)

**Comment by pdaleramirez** [comment]
@BenParizek i have modified the previous behavior where subscriber can only subscribe to one list. We may want a setting where user can allow multiple or single subscription. Commited here https://github.com/barrelstrength/craft-sprout-lists/commit/acae20792dbd82e95eb66b0c0b675b6378d4fae2

**Comment by BenParizek** [comment]
@katiasmet This should be resolved in v1.0.0-beta.14
